framework/include/task_manager/task_manager.h

```diff
@@ -178,6 +179,18 @@ int task_manager_pause(int handle, int timeout);
  * @since TizenRT v2.0 PRE
  */
 int task_manager_resume(int handle, int timeout);
+/**
+ * @brief Request to restart the task
```

Comment:
please add description that restart cannot guarantee the memory deallocation

Reply:
Done